packages/edit-site/src/components/sidebar-navigation-screen-templates-browse/index.js

```diff
@@ -10,14 +12,24 @@ import SidebarNavigationScreen from '../sidebar-navigation-screen';
 import DataviewsTemplatesSidebarContent from './content';
 
 export default function SidebarNavigationScreenTemplatesBrowse( { backPath } ) {
+	const isBlockBasedTheme = useSelect(
+		( select ) => select( coreStore ).getCurrentTheme()?.is_block_theme,
+		[]
+	);
 	return (
 		<SidebarNavigationScreen
 			title={ __( 'Templates' ) }
-			description={ __(
-				'Create new templates, or reset any customizations made to the templates supplied by your theme.'
-			) }
+			description={
+				isBlockBasedTheme
+					? __(
+							'Create new templates, or reset any customizations made to the templates supplied by your theme.'
+					  )
+					: null
+			}
```

Comment:
```suggestion
			description={
				isBlockBasedTheme &&
				__(
					'Create new templates, or reset any customizations made to the templates supplied by your theme.'
				)
			}
```

Regarding the `content` prop, I think it can be simplified by using `&&`.